src/MBrace.Azure/paket.template

```diff
@@ -36,15 +36,6 @@ files
 	../../bin/Microsoft.ServiceBus.dll ==> tools
 	../../bin/Microsoft.WindowsAzure.Configuration.dll ==> tools
 	../../bin/Microsoft.WindowsAzure.Storage.dll ==> tools
-	../../bin/Hyak.Common.dll ==> tools
```

Comment:
Yes. The "tools" folder exists merely for the standalone "mbrace.azureworker.exe" to function. Since MBrace.Azure.Management is just a class library, we can load all relevant dependencies using a script file, which I've added to the base of the nuget package.